python/orca/example/learn/tf/transfer_learning/transfer_learning.py

```diff
@@ -48,24 +48,30 @@
 parser = argparse.ArgumentParser()
 parser.add_argument('--cluster_mode', type=str, default="local",
                     help='The mode for the Spark cluster. local, yarn or spark-submit.')
-parser.add_argument('--data_dir', type=str, default="./dataset", help='The path of datesets.')
+#parser.add_argument('--data_dir', type=str, default="./dataset", help='The path of datesets.')
 parser.add_argument('--batch_size', type=int, default=64, help='The training batch size')
 parser.add_argument('--epochs', type=int, default=2, help='The number of epochs to train for')
+#dataset path
+parser.add_argument('--file_path', type=str, default="tmp/data/",
+                    help="The path to cats_and_dogs_filtered.zip.")
 args = parser.parse_args()
 cluster_mode = args.cluster_mode
 
-dataset_dir = args.data_dir
-if not exists(dataset_dir):
-    makedirs(dataset_dir)
-zip_file = tf.keras.utils.get_file(
-    origin="https://storage.googleapis.com/mledu-datasets/cats_and_dogs_filtered.zip",
-    fname="cats_and_dogs_filtered.zip", extract=True, cache_dir=dataset_dir)
-base_dir, _ = os.path.splitext(zip_file)
+#dataset_dir = args.data_dir
```

Comment:
You should not just comment out a line. If you do not want them, just delete them.

Reply:
OK. I will modify it.